src/cmd-buildextend-metal

```diff
@@ -128,7 +128,7 @@ size="$(jq '."estimate-mb".final' "$PWD/tmp/ostree-size.json")"
 # extra size is the non-ostree partitions, see create_disk.sh
 size="$(( size + 513 ))M"
 echo "Disk size estimated to $size"
-kargs="$(python3 -c 'import sys, yaml; args = yaml.safe_load(sys.stdin)["extra-kargs"]; print(" ".join(args))' < "$configdir/image.yaml")"
+kargs="$(python3 -c 'import sys, yaml; args = yaml.safe_load(sys.stdin); print(" ".join(args.get("extra-kargs", "")))' < "$configdir/image.yaml")"
```

Comment:
Let's split out this patch into a separate PR, it looks clearly right.

Also let's add a commit message comment like:

> RHCOS doesn't set `extra-kargs` in the `image.yaml`.